Cargo.toml

```diff
@@ -15,9 +15,14 @@ members = [
   "pcli",
   "pcli-next",
   "tct",
+  "tct-property-test",
   "ibc",
 ]
 
+[profile.test]
+# Maximum optimization level for tests, so that they are fast (especially the proptest ones)
+opt-level = 3
```

Comment:
This is changing the entire workspace configuration to do more expensive builds, regardless of whether or not the expensive tests are actually going to be run.

Could we just do `cargo test --release` instead when running the slow, long-running property tests?